### ISSUE TITLE
VectorTile layer not displayed until I zoom

### ISSUE THREAD

**Issue by Akbalder** [openlayers/openlayers#16456]
**Describe the bug**
My VectorTile layer isn't displayed when I load the map. I have to zoom to display it. After that, it functions correctly.
I have the issue with OpenLayers v10.3.0 and v10.3.1. It functions correctly with OpenLayers v10.2.1.

**To Reproduce**
I didn't find what causes the issues. I didn't reproduce it outside my project. I guess that it is linked to a specific sequence.
The layer not displayed is just a VectorTile layer using a VectorTile source.

**Expected behavior**
The layer should be displayed directly as in previous versions of OpenLayers.

**Comment by ahocevar** [comment]
Maybe an incorrect tile grid configuration? Please share a minimal example that shows the problem.

**Comment by Akbalder** [comment]
I removed most of my configuration but I still have the issue.
```
olLayer = new OlLayerVectorTile({
    source: new OlSourceVectorTile<OlFeature>({
        format: new OlFormatGeoJSON(),
        url: 'https://localhost:8080/data/tiles/{z}-{x}-{y}.geojson'
    }),
    zIndex: 99999999,
});
```

The same code functions correctly with OpenLayers v10.2.1 so I think that it is linked to a modification made on OpenLayers.

**Comment by Akbalder** [comment]
I found what's causing it: A call to source.refresh().
On https://codesandbox.io/p/sandbox/tile-layer-not-displayed-hfh853?file=%2Fmain.js, we have to zom to see the VectorTileLayer.

**Comment by mike-000** [comment]
> A call to source.refresh().

That is a similar problem to #16444 and #16455 also seems to fix it

**Comment by ahocevar** [comment]
Fixed with #16455
